thunderbird/utils.py

```diff
@@ -52,4 +53,4 @@ def get_url():
         if url:
             return url
 
-    return "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps"
+    return "https://marble-dev01.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps"
```

Comment:
Again, it would be best if this were obtained from an env var or other external source.